.github/workflows/nodejs.condo.ci.yml

```diff
@@ -304,3 +278,63 @@ jobs:
           node bin/wait-apps-apis.js -f condo condorb) && \
           yarn workspace @app/condorb test"
 
+  run-dev-api-tests:
+    name: DEV-API Tests
+    runs-on: ubuntu-latest
+    needs:
+      - detect-changes
+      - build-image
+    if: ${{ needs.detect-changes.outputs.dev-api == 'true' }}
+    steps:
+      - name: Login to cloud registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ secrets.DOCKER_REGISTRY }}
+          username: ${{ secrets.SBERCLOUD_CR_USERNAME }}
+          password: ${{ secrets.SBERCLOUD_CR_PASSWORD }}
+      - name: Setup PG db
+        run: |
+          docker run -e POSTGRES_USER=$POSTGRES_USER -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_DB=$POSTGRES_DB -p="127.0.0.1:5432:5432" -d ${{ env.PG_IMAGE_FULL }}
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: main
+      - name: Setup Redis db
+        run: |
+          docker run -p="127.0.0.1:6379:6379" -d ${{ env.REDIS_IMAGE_FULL }}
```

Comment:
On `ubuntu-latest` you can run compose services for now. So you don't need to manually create db services

Reply:
But I need to checkout all repo for this reason...Idk, maybe we can refactor it later by inventing something better